### PR TITLE
Updated EventSchedule to EventGroupings

### PR DESCRIPTION
**Pull request by asleepingpanda**
Updates the UI for Events to display the Schedule properly according to the new EventGroupings schema that is seen in The Big Merge

This PR is dependent on [The Big Merge](https://github.com/christfellowshipchurch/apollos-api/pull/144) being set live

### CODE REVIEW

**Comment by jordanwade** [comment]
@asleepingpanda I'm seeing a discrepancy between times on the `cf-young-adults` event. Is that related to these changes? 
image

**Comment by asleepingpanda** [comment]
Aw man.. this is a Daylight Savings error. Basically right now, the current time offset is -4 hours (for EST) but then that offset changes to be -5 hours when Daylight Savings happens.

Since this event is post-daylight savings, it's displaying the offset as -5 instead of -4 for the future event. Once Daylight Savings passes, this will go back to the correct time.

Idk how to fix it... I've spent more hours than I know trying to resolve Daylight Savings and Time Zone issues and this one still exists.

**Comment by jordanwade** [comment]
@asleepingpanda I think I've got it now. Looks like changing the campus location from the dropdown updated the schedule date. If that is what you wanted to smoke test ✅ 
image

File: src/content-single/EventContentItem/EventDetail.js [hunk 1]
```diff
@@ -39,64 +40,58 @@ const EventDetail = ({
...
-          'align-items-center',
-          'pb-3'
-        )}
-      >
-        <div className="mt-4 mb-2 pb-2">
-          <LiveIndicator isLive={isLive} />
-          <h1 className="mb-2 text-dark">{title}</h1>
-          <h3 className="mt-1 content-subtitle font-weight-light">{summary}</h3>
+}) =>
+  console.log({ id }) || (
```
Comment (jordanwade) [inline src/content-single/EventContentItem/EventDetail.js:44]: kill console.log

File: src/content-single/EventContentItem/EventGroupings/index.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  defaultSelection: null,
+};
+
+const EventGroupingsConnected = ({ contentId }) => {
+  const { data, loading, error } = useQuery(GET_EVENT_GROUPINGS, {
+    variables: { id: contentId },
+    skip: !contentId || contentId === '',
+  });
+
+  console.log({ data });
```
Comment (jordanwade) [inline src/content-single/EventContentItem/EventGroupings/index.js:164]: kill console.log

File: src/content-single/EventContentItem/EventLayout.js [hunk 3]
```diff
@@ -11,7 +11,11 @@ import {
   EventSchedule,
 } from './components';
 
-const EventLayout = ({ content }) => {
+import EventGroupings from './EventGroupings';
+
+const EventLayout = ({ contentId, content }) => {
+  console.log({ content });
```
Comment (jordanwade) [inline src/content-single/EventContentItem/EventLayout.js:17]: kill console.log

**Comment by jordanwade** [comment]
@asleepingpanda besides a few console.logs looks good to me.

**Review by jordanwade — APPROVED** [review]

**Pull request merged**